client/src/modules/employees/registration/employees.js

```diff
@@ -1,43 +1,62 @@
 // TODO Handle HTTP exception errors (displayed contextually on form)
 angular.module('bhima.controllers')
-.controller('EmployeeController', EmployeeController);
+  .controller('EmployeeController', EmployeeController);
 
 EmployeeController.$inject = [
   'EmployeeService', 'ServiceService', 'GradeService', 'FunctionService',
-  'CreditorGroupService', 'util', 'NotifyService','$state',
-  'bhConstants', 'ReceiptModal', 'SessionService',
+  'CreditorGroupService', 'util', 'NotifyService', '$state',
+  'bhConstants', 'ReceiptModal', 'SessionService', 'PatientService',
 ];
 
-function EmployeeController(Employees, Services, Grades, Functions, CreditorGroups, util, Notify, $state, bhConstants, Receipts, Session) {
+function EmployeeController(Employees, Services, Grades, Functions, CreditorGroups, util, Notify, $state, bhConstants, Receipts, Session, Patients) {
   var vm = this;
   var referenceId = $state.params.id;
+  var saveAsEmployee = $state.params.saveAsEmployee;
+  var isUpdating = !!$state.params.id && !saveAsEmployee;
 
   vm.enterprise = Session.enterprise;
-  vm.isUpdating = $state.params.id ? true : false;
 
   vm.origin = '';
 
-  if (referenceId) {
+  if (referenceId && !saveAsEmployee) {
     Employees.read(referenceId)
-    .then(function (employee) {
-      formatEmployeeAttributes(employee);
-      vm.origin = employee.hospital_no;
-      vm.employee = employee;
-    })
-    .catch(function (error) {
+      .then(function (employee) {
+        formatEmployeeAttributes(employee);
+        vm.origin = employee.hospital_no;
+        vm.employee = employee;
+      })
+      .catch(function (error) {
+      // handle error and update view to show no results - this could be improved
+        Notify.handleError(error);
+        vm.unknownId = true;
+      });
+  }
 
+  if (saveAsEmployee) {
+    Patients.read(referenceId)
+      .then(function (patient) {
+        vm.employee.display_name = patient.display_name;
+        vm.employee.dob = new Date(patient.dob);
+        vm.employee.sex = patient.sex;
+        vm.employee.hospital_no = patient.hospital_no;
+        vm.employee.is_patient = true;
+        vm.employee.patient_uuid = patient.uuid;
+        vm.employee.debtor_uuid = patient.debtor_uuid;
```

Comment:
We also need to put the `debtor_group_uuid` someplace.  For some reason, it is not being populated when the page loads.